implementations/contracts/ERC725XCore.sol

```diff
@@ -130,6 +130,13 @@ abstract contract ERC725XCore is OwnableUnset, ERC165, IERC725X {
         uint256[] memory values,
         bytes[] memory datas
     ) internal virtual returns (bytes[] memory) {
+        if (
```

Comment:
Maybe we can move this check below the check of `ERC725X_ExecuteParametersLengthMismatch` and then we can check just one parameter that is equal to 0.

Reply:
love your suggested comment! 🎉